scripts/ci/test_selection_adjacency.yaml

```
@@ -65,11 +65,16 @@ adjacency:
   # that every src/omnibase_core/<dir> has an entry.
   adapters: {reverse_deps: []}
   agents: {reverse_deps: []}
+  artifacts: {reverse_deps: []}
   analysis: {reverse_deps: []}
```

YAMLlint (1.37.1)

[error] duplication of key "analysis" in mapping

(key-duplicates)